core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -360,6 +361,168 @@ class AdminClientIntegrationTest extends KafkaServerTestHarness with Logging {
   }
 
   @Test
+  def testCreatePartitions(): Unit = {
+    client = AdminClient.create(createConfig)
+
+    // Create topics
+    val topic1 = "create-partitions-topic-1"
+    TestUtils.createTopic(zkUtils, topic1, 1, 1, servers, new Properties)
+
+    // assert that the topic has 1 partition
+    assertEquals(1, client.describeTopics(Set(topic1).asJavaCollection).values.get(topic1).get.partitions.size)
+
+    val validateOnly = new CreatePartitionsOptions().validateOnly(true)
+
+    // assert that a validateOnly request doesn't increase the number of partitions
+    var alterResult = client.createPartitions(Map(topic1 -> NewPartitions.increaseTo(new Integer(2))).asJava, validateOnly)
```

Comment:
Nit: a few long lines in this test. Also, no need for `new Integer` given auto-boxing.